src/main/scala/cromwell/engine/db/slick/WorkflowExecutionComponent.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.engine.db.slick
+
+import java.sql.Timestamp
+import java.util.UUID
+
+import slick.dbio.{NoStream, Effect}
+
+case class WorkflowExecution
+(
+  workflowExecutionUuid: String,
+  wdlUri: String,
+  status: String,
+  startDt: Timestamp,
+  workflowExecutionId: Option[Int] = None,
+  endDt: Option[Timestamp] = None
+  )
+
+trait WorkflowExecutionComponent {
+  this: DriverComponent =>
+
+  import driver.api._
+
+  class WorkflowExecutions(tag: Tag) extends Table[WorkflowExecution](tag, "WORKFLOW_EXECUTION") {
+    def workflowExecutionId = column[Int]("WORKFLOW_EXECUTION_ID", O.PrimaryKey, O.AutoInc)
+
+    def workflowExecutionUuid = column[String]("WORKFLOW_EXECUTION_UUID")
+
+    def wdlUri = column[String]("WDL_URI")
+
+    def status = column[String]("STATUS")
+
+    def startDt = column[Timestamp]("START_DT")
+
+    def endDt = column[Timestamp]("END_DT")
+
+    override def * = (workflowExecutionUuid, wdlUri, status, startDt, workflowExecutionId.?, endDt.?) <>
+      (WorkflowExecution.tupled, WorkflowExecution.unapply)
+  }
+
+  val workflowExecutions = TableQuery[WorkflowExecutions]
+
+  val workflowExecutionsAutoInc = workflowExecutions returning workflowExecutions.
```

Comment:
Is this The Slick Way of doing things, A Slick Way or A Way? I'm going to have to take your word for it either way but curious how codified these things are.